t2kdm/backends.py

```diff
@@ -885,13 +884,13 @@ def _replicas(self, lurl, **kwargs):
         self._check_return_value(rep)
         rep = rep["Value"]["Successful"][lurl]
 
-        return rep.values()
+        return list(rep.values())
 
     def _exists(self, surl, **kwargs):
         try:
             ret = self._ls_se_cmd(surl, "-d", "-l", **kwargs).strip()
         except sh.ErrorReturnCode as e:
-            if "No such file" in e.stderr:
+            if "No such file" in str(e):
```

Comment:
With my setup, I cannot get stderr from e; only a long string which I just print for debugging purpose: not ideal...

Reply:
What version of the python `sh` package do you have?

Reply:
@ast0815 I am using sh version 1.14.2 in the container

Reply:
Pypi says this is the latest version